libs/ui-kit/spinner/spinner.component.stories.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable import/no-default-export */
+import { Story, Meta } from '@storybook/angular';
+import { CommonModule } from '@angular/common';
+import { SpinnerComponent } from './spinner.component';
+
+export default {
+  title: 'SpinnerComponent',
+  component: SpinnerComponent,
+  argTypes: {
+    size: {
+      name: 'Size ("--size" css variable)',
+      description: 'Size of the spinner.',
+      defaultValue: '20px',
+      control: {
+        type: 'text',
+      },
+    },
+    color: {
+      name: 'Color ("--color" css variable)',
+      description: 'Forefront color of the spinner.',
+      defaultValue: 'var(--cz-color-primary, #007bff)',
+      control: {
+        type: 'color',
+      },
+    },
+    bgColor: {
+      name: 'Background Color ("--color-background" css variable)',
+      description: 'Background color of the spinner.',
+      defaultValue: '#ccc',
```

Comment:
maybe we can use `var` here too? 🤔

Reply:
For now, I don't think that a globally defined var would make sense, as it's  acolor specific to the spinner I would say 🤔